<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-radius: 10px;</string>
  </property>
  <widget class="QPushButton" name="logoButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>200</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose Logo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="generateButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>400</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="importButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Import CSV</string>
   </property>
  </widget>
  <widget class="QPushButton" name="outputButton">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>400</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Output</string>
   </property>
  </widget>
  <widget class="QLabel" name="logoPreview">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="qrPreview">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>260</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QTableWidget" name="loadedItems">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>341</width>
     <height>331</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>URL</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
